src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -2011,6 +2011,7 @@ private void getCallInfo(ref CORINFO_RESOLVED_TOKEN pResolvedToken, CORINFO_RESO
             {
                 // Delegate targets are always treated as direct calls here. (It would be nice to clean it up...).
                 directCall = true;
+                resolvedCallVirt = !targetMethod.IsVirtual || targetMethod.IsFinal || targetMethod.OwningType.IsSealed();
```

Comment:
There is comment above about getting this cleaned up, and the new line is duplicating condition from a couple of lines below.

Would it be better to get rid of this special treatment and move the check for LDFTN to be later when we are actually creating the entrypoint for JIT to use?